src/ducks/stripeConnectAccount.duck.js

```diff
@@ -226,11 +224,12 @@ export const createStripeAccount = params => (dispatch, getState, sdk) => {
 // https://www.sharetribe.com/api-reference/?javascript#update-stripe-account
 export const updateStripeAccount = params => (dispatch, getState, sdk) => {
   const { bankAccountToken } = params;
+  const bankAccountTokenMaybe = bankAccountToken ? { bankAccountToken } : {};
 
   dispatch(stripeAccountUpdateRequest());
   return sdk.stripeAccount
     .update(
-      { bankAccountToken, requestedCapabilities: ['card_payments', 'transfers'] },
+      { ...bankAccountTokenMaybe, requestedCapabilities: ['card_payments', 'transfers'] },
```

Comment:
I'm not sure it makes sense to keep this optional parameter around.

https://www.sharetribe.com/api-reference/marketplace.html#update-stripe-account

Reply:
Removed!